orchagent/switchorch.h

```diff
@@ -12,20 +12,21 @@ struct WarmRestartCheck
 class SwitchOrch : public Orch
 {
 public:
-    SwitchOrch(swss::DBConnector *db, std::string tableName);
-
+    SwitchOrch(swss::DBConnector *db, std::string tableName, TableConnector switchTable);
```

Comment:
Looks like the change is missed in constructor updating `m_switchTable`

Reply:
Thanks @prsunny .  Missed the update. Fixed now